packages/notification-spa/src/app/manage-notification/manage-notification.component.ts

```diff
@@ -0,0 +1,135 @@
+import { Component, OnInit } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { AppService } from '../app.service';
+import { UserProfile } from '../helper';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'op-manage-notification',
+  templateUrl: './manage-notification.component.html',
+  styleUrls: ['./manage-notification.component.scss']
+})
+export class ManageNotificationComponent implements OnInit {
+  user = UserProfile;
+  days: Day[] = [
+    {
+      day: 'mon',
+      status: false,
+    },
+    {
+      day: 'tue',
+      status: false,
+    },
+    {
+      day: 'wed',
+      status: false,
+    },
+    {
+      day: 'thurs',
+      status: false,
+    },
+    {
+      day: 'fri',
+      status: false,
+    },
+    {
+      day: 'sat',
+      status: false,
+    },
+    {
+      day: 'sun',
+      status: false,
+    },
+  ];
+
+  events: EventType[] = [
+    {
+      name: 'create',
+      status: false,
+    },
+    {
+      name: 'update',
+      status: false,
+    },
+    {
+      name: 'delete',
+      status: false,
+    },
+  ];
+
+  applications = [
+    {
+      name: 'Outage Management',
+    },
+    {
+      name: 'TS Catalog',
+    },
+    {
+      name: 'Outage Management',
+    },
+  ];
+  channel: string;
+  type: string;
+  template: string;
+  schedule: Schedule;
+  createdBy: string;
+  createdOn: Date;
```

Comment:
Date should be should be defined in UTC

Reply:
Removed the dummy data and added new Date().toUTCString()